app/main/views/marketplace.py

```diff
@@ -118,7 +116,11 @@ def list_opportunities(framework_framework):
 
     api_result = form.get_briefs()
 
-    briefs = api_result["briefs"]
+    briefs = [{
```

Comment:
I'd say that ideally everything needed to display a brief should be in the "display brief" manifest, and that would be the only thing passed to the template. For services I know there are some "fake" questions in the frameworks content, never rendered as form questions but just there for display manifests (e.g. lotName I think). Maybe something like that could work here?  Or is everything in the manifest going to appear in the summary table part of the brief page and so extra fake questions will make a mess of that?

I'm not 100% against this as it is, but it does feel like a proliferation of slightly different "special" ways of doing things, which it would be nice to avoid where possible.